rpc/wordvault/api.proto

```diff
@@ -30,9 +31,13 @@ message GetCardInfoRequest {
 message GetNextScheduledRequest {
   string lexicon = 1;
   uint32 limit = 2;
+  optional uint64 deck_id = 3;
 }
 
-message GetSingleNextScheduledRequest { string lexicon = 1; }
+message GetSingleNextScheduledRequest {
+    string lexicon = 1;
+    optional uint64 deck_id = 2;
```

Comment:
What do you think of using the zero-value of deck_id instead of optional? I've never really been a big fan of proto bringing back `optional` but I don't know how dogmatic I am about it...

I guess partially it's the dealing with pointers etc. Let me know what you think.

Reply:
No preference from me! will switch it around

Reply:
Done! Seems like I didn't need to update the tests (non-present keys seem to get parsed as 0 in go), let me know if I'm off-base there because if so the tests need to be improved

Reply:
Yeah I think that's fine. Go has the concept of zero values for uninitialized variables. It plays nicely with proto in this way.